testplan/runnable/interactive/base.py

```diff
@@ -482,9 +480,9 @@ def add_created_environment(self, env_uid):
         """Add an environment from the created environment maker instance."""
         self.target.add_environment(self._created_environments[env_uid])
 
-    def reload(self, rebuild_dependencies=True):
+    def reload(self, rebuild_dependencies=False):
```

Comment:
Is dependencies built the first time reload is called? If we set rebuild_dependencies default False, when are we going to build dependencies?

Reply:
Dependencies are built when the reloader object is constructed. Generally you don't want to rebuild dependencies every time, because for a large project that could take a while (~seconds) especially on an NFS.

Reply:
The only time you should need to reload dependencies is if you add (or delete) an entire module, which is not the common use-case where someone just tweaks a few existing testcases.